openstack/containerinfra/v1/clustertemplates/requests.go

```diff
@@ -123,3 +124,51 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 	}
 	return
 }
+
+type UpdateOpts struct {
+	Path  string `json:"path" required:"true"`
+	Value string `json:"value,omitempty"`
+	Op    string `json:"op" required:"true"`
```

Comment:
Let's create a type for `Op`. It looks like Zaqar Queues implements a similar API pattern, so we can steal from what we've done in Gophercloud: https://github.com/gophercloud/gophercloud/blob/master/openstack/messaging/v2/queues/requests.go#L134-L147

Reply:
Sounds good